dnf/cli/output.py

```diff
@@ -1207,8 +1216,8 @@ def _fits_in_cols(msgs, num):
         list_bunch = _make_lists(transaction, self.base._goal)
 
         for (action, tsis) in [(_('Reinstalled'), list_bunch.reinstalled),
-                               (_('Removed'), list_bunch.erased +
-                                list_bunch.erased_clean),
+                               (_('Removed'), list_bunch.erased + list_bunch.erased_dep
+                                + list_bunch.erased_clean),
```

Comment:
@j-mracek the code looks good to me except the flake8 issue reported by dnf-bot. Could you fix it?